site/en/docs/extensions/reference/tabCapture/index.md

```diff
@@ -3,4 +3,61 @@ api: tabCapture
 has_warning: This permission <a href="/docs/extensions/mv3/permission_warnings/#permissions_with_warnings">triggers a warning</a>.
 ---
 
-<!-- Intentionally blank -->
+## Overview
+
+The chrome.tabCapture API allows you to access a [MediaStream][media-stream] containing video and
+audio of the current tab. It can only be called after the user invokes an extension by clicking or
+otherwise triggering the extension's action button, similar to the behavior of the
+[activeTab][active-tab] permission.
```

Comment:
>invokes an extension by clicking...

I think it goes without saying that clicking an action button invokes an extension

>or otherwise triggering the extension's action button

Is there some other way to trigger a button?

Reply:
Yep, it can happen based on a keyboard shortcut or tabbing to the UI and hitting the space bar. But I thought this was the best compromise between hinting at there being different ways and not getting in to the weeds of all of the possibilities, and then "it won't happen if you call `chrome.action.disable`" etc etc...

Reply:
I've done the best I can making this clearer. Let me know what you think.